src/write.c

```diff
@@ -1831,11 +1831,24 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
     }
 
     // -----------------------------------------------------------------------
-    // Map quality and qualityAlpha to quantizer and quantizerAlpha
+    // Map quality settings to quantizer values.
```

Comment:
Optional: I suggest moving the changes to codec_aom.c to a separate PR and keeping only the changes to this file for the v1.2.0 release.

I assume the changes to this file are sufficient to accomplish the goal of the PR: Make alpha and gain map quality default to the same as color in API.